test_scripts/API/SetMediaClockTimer/008_TRACK_Indicator_type_with_seektime_INVALID_DATA.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
i'm not sure about rejecting rpcs with extraneous values, what if in the future we do decide to use this for some reason? I don't want to force mobile to add logic to not send this param

Reply:
I agree with your point about not rejecting rpcs with extraneous values but in this case I think we should since the values are incompatible.

The seek indicator type TRACK is incompatible with the seekTime parameter since TRACK would skip directly to the next/previous track regardless of the value in the seekTime param. Having a seek duration in this case wouldn't really make sense.